mypy/checkexpr.py

```diff
@@ -1886,6 +1886,13 @@ def infer_literal_expr_type(self, value: LiteralValue, fallback_name: str) -> Ty
                 column=typ.column,
             ))
 
+    def concat_tuples(self, left: TupleType, right: TupleType) -> TupleType:
+        """Concatenate two fixed length tuples."""
+        result = TupleType(items=left.items + right.items,
+                           fallback=self.named_type('builtins.tuple'))
+        result.partial_fallback = tuple_fallback(result)
```

Comment:
This line seems unnecessary. Anybody who needs the fallback type should just call `tuple_fallback` instead of assuming that `partial_fallback` has the correct type.

Reply:
Thanks! Removed the line.